packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -1053,6 +1139,29 @@ class _CupertinoTimerPickerState extends State<CupertinoTimerPicker> {
     );
   }
 
+  /// Sets the picker to be the initial time.
+  void _desiredDurationChanged() {
+    final Duration desiredTimerDuration = widget.controller.desiredTimerDuration;
+    final Duration animationDuration = widget.controller.resetAnimationDuration;
```

Comment:
This should be a property of this widget rather than of the controller.

Reply:
Just the animation duration right?